src/Common/BackspaceIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import './BackspaceIcon.css';
+
+class BackspaceIcon extends React.Component {
+  public render() {
```

Comment:
You could have just used U+232B.  :P 

https://www.fileformat.info/info/unicode/char/232b/index.htm

You don't necessarily have to get rid of this - it works fine - but in my experience there are font glyphs for a lot of things.

Reply:
Though one argument in favor would be that if we ever decided to add RTL support, font glyphs are easy to substitute - just replace with U+2326 in this case instead of having to draw a new SVG.